src/main/java/mil/dds/anet/config/AnetConfiguration.java

```diff
@@ -146,6 +183,57 @@ public void setDictionary(Map<String, Object> dictionary) {
     this.dictionary = Collections.unmodifiableMap(dictionary);
   }
 
+  public void loadDictionary() throws IOException, IllegalArgumentException {
+    // Read and set anet-dictionary
+    final File file = new File(System.getProperty("user.dir") + "/" + getAnetDictionaryName());
```

Comment:
Line is better written as:
```
    // scan:ignore
    final File file = new File(System.getProperty("user.dir"), getAnetDictionaryName());
```
to avoid using platform-specific path separators (and to suppress the false positive from ShiftLeft scan).

Reply:
PR description is updated